lib/ts/station/FSM.ex

```diff
@@ -79,6 +82,7 @@ defmodule Station.FSM do
     # When an itinerary is passed to the station
     defevent query_input(itinerary),
       data: station_data = [_station_struct, _dependency] do
+      Logger.info(fn -> "Current state='ready'; Next state='query_rcvd'" end)
```

Comment:
This will be info message, `fn -> "query <query_id> received at station <station_name/id>"`

Both query_id and station values need to be found and added to the info message string.